ammonite/httpMethods.sc

```diff
@@ -126,6 +126,35 @@ object Test {
                  ).get,
        keyChain = List(cosyKey,localKey)
    )
+   
+   //----------------------------------------------------------------------------------------------
+
+   def deleteLocalKey = http.run(
+     req      = DELETE[`text/turtle`](
+                  resource = Uri("https://localhost:8443/2013/")
+                )
+   )
+
+   def getLocalKey = http.run(
+      req     = GETpg(
+                  uri = Uri("https://localhost:8443/2013/"),
+                  keyChain = List(localKey)
+     )
+   )
+
+   def putLocalKey = http.run(
+     req      = PUT[`text/turtle`](
+                  resource = Uri("https://localhost:8443/2013/"),
+                  graph    = pubKeyPG.get.graph
+                ).get,
+     keyChain = List(localKey)
+   )
```

Comment:
here you are trying to PUT a resource on the container. POST creates a new resource. But PUT overwrites the one given by the URL. Do you actually want to overwrite the container?